app/controllers/admin/accessories_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Admin
+  class AccessoriesController < BaseController
+
+    def create
+      @accessory = Accessory.new(accessory_params)
+      if @accessory.save
+        flash[:success] = "You have created #{@accessory.title}!"
+        redirect_to bike_shop_path
+      else
+        render :'admin/bike_shop/new'
```

Comment:
Huh, I don't normally see it like this. Perhaps we could change to an admin accessory new path?